src/lib/import-export/import/importers/importer.ts

```diff
@@ -308,3 +314,36 @@ export class SQLImporter extends BaseImporter {
 		}
 	}
 }
+
+export class WpressImporter extends BaseBackupImporter {
+	protected async parseMetaFile(): Promise< MetaFileData | undefined > {
+		return undefined;
+	}
+
+	protected async prepareSqlFile( tmpPath: string ): Promise< void > {
+		const tempOutputPath = `${ tmpPath }.tmp`;
+		const readStream = createReadStream( tmpPath, 'utf8' );
+		const writeStream = createWriteStream( tempOutputPath, 'utf8' );
+
+		const rl = createInterface( {
+			input: readStream,
+			crlfDelay: Infinity,
+		} );
+
+		rl.on( 'line', ( line: string ) => {
+			writeStream.write( line.replace( /SERVMASK_PREFIX/g, 'wp' ) + '\n' );
```

Comment:
Do you refer that we should add a comment explaining why why we replace this fake prefix?
It seems that wpress replaces the original prefix with SERVMASK_PREFIX, probably to avoid prefix collisions when importing to a shared database.

Reply:
> Do you refer that we should add a comment explaining why why we replace this fake prefix? It seems that wpress replaces the original prefix with SERVMASK_PREFIX, probably to avoid prefix collisions when importing to a shared database.

Yep, I was curious why we need this. Thanks @sejas for explaining it 🙇 !